scripts/local_environment.sh

```diff
@@ -18,21 +18,33 @@ function usage() {
     ----------
     --deploy        - Perform creation of test container
     --destroy       - Perform destroy of test container
-    --get-runtime   - Get available container runtime
+    --runtime       - Sets runtime to be used.
+                      Supported args: docker/podman
+                      By default - "docker"
     --help|-h       - Print help
 EOF
 }
 
-function check_container_engine() {
-    if command -v podman &> /dev/null; then
-	RUNTIME=podman
-    elif command -v docker &> /dev/null; then
-        RUNTIME=docker
+function sets_container_engine() {
```

Comment:
This only checks whether the chosen container engine is actually available, so

```suggestion
function check_container_engine() {
```

still seems appropriate.

Reply:
ack.